proto/README.md

```diff
@@ -22,32 +28,42 @@ Note that each proto package is versioned, but this version is
 independent of any API version. It would be possible for a single
 API version to support multiple event versions, or vice versa.
 
-Our convention for event types is to have a dot-separated list of:
+Our convention for event types is to have a dot-separated list of
+the following values, in the order:
 
 - `google` (always)
 - product (e.g. `firebase` or `cloud.storage`)
 - resource (e.g. `object` or `document`)
 - version (e.g. `v1`)
-- the action causing the event to be emitted (e.g. `finalized`, `deleted`)
+- action (e.g. `finalized`, `deleted`)
```

Comment:
what does `,` mean here whereas the other items have `or`?

Reply:
Fixed.